contracts/0.8.25/utils/OZPausableUntil.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+// See contracts/COMPILERS.md
+pragma solidity 0.8.25;
+
+import {PausableUntil} from "contracts/common/utils/PausableUntil.sol";
+import {AccessControlEnumerableUpgradeable} from "contracts/openzeppelin/5.0.2/upgradeable/access/extensions/AccessControlEnumerableUpgradeable.sol";
+
+/// @title OZPausableUntil is a PausableUntil reference implementation using OpenZeppelin's AccessControlEnumerableUpgradeable
+/// @author folkyatina
+abstract contract OZPausableUntil is PausableUntil, AccessControlEnumerableUpgradeable {
```

Comment:
OZ does not describe the ACL functions of the contract; maybe PermissionedPausableUntil / ManagablePausableUntil?